modules/virt-creating-vm-snapshot-cli.adoc

```diff
@@ -64,7 +64,7 @@ $ kubectl wait my-vm my-vmsnapshot --for condition=Ready
 +
 [NOTE]
 ====
-Online snapshots have a default time deadline of five (`5`) minutes. If the snapshot does not complete successfully in five minutes, the status is set to `failed`. Afterwards, the file system will be thawed and the VM unfrozen but the status remains `failed` until you delete the failed snapshot image.
+Online snapshots have a default time deadline of three minutes (`3m`). If the snapshot does not complete successfully in three minutes, the status is set to `failed`. Afterwards, the file system will be thawed and the VM unfrozen but the status remains `failed` until you delete the failed snapshot image.
```

Comment:
I guess you missunderstand me. the default is 5minutes. the correction needed is in line 69 that the attribute is not (in minutes)

Reply:
Complete note revised:

_Online snapshots have a default time deadline of five minutes (`5m`). If the snapshot does not complete successfully in three minutes, the status is set to `failed`. Afterwards, the file system will be thawed and the VM unfrozen but the status remains `failed` until you delete the failed snapshot image.

To change the default time deadline, add the `FailureDeadline` attribute to the VM snapshot spec with the time designated in minutes (`m`) that you want to specify before the snapshot operation times out.

To set no deadline, you can specify `0`, though this is generally not recommended, as it can result in an unresponsive VM._